src/ucp/core/ucp_worker.c

```diff
@@ -1587,7 +1587,7 @@ char *ucp_worker_print_used_tls(const ucp_ep_config_key_t *key,
     p += strlen(p);
 
     for (lane = 0; lane < key->num_lanes; ++lane) {
-        if ((key->am_lane == lane) ||
+        if ((key->am_lane == lane) || (key->rkey_ptr_lane == lane) ||
```

Comment:
better make it printed explicitly as rkey_ptr lane

Reply:
but we do not print it by lane types, we print it by features:
`[1628016549.696474] [swx-dgx03:75816:0] ucp_worker.c:1810 UCX INFO ep_cfg[0]: tag(posix/memory xpmem/memory cma/memory);`

Reply:
ok